raysort/main_push.py

```diff
@@ -20,6 +21,9 @@
 from raysort.typing import PartId, PartInfo
 
 
+MergeStats = namedtuple("MergeStats", "datachunk_sizes total_bytes")
```

Comment:
Did you mean `namedtuple("MergeStats", ["datachunk_sizes", "total_bytes"])`?

Also, move this to `typing.py`.

Reply:
actually both formats work, but I can switch to the one you suggested